src/repositories/AdminRepository.ts

```diff
@@ -0,0 +1,16 @@
+import {getCollection} from "./connectToDB";
+import {Admin} from "./schemas";
+
+export default {
+  addAdmin(username: string, passwordHash: string) {
+    getCollection("admin").insertOne({username, passwordHash});
+  },
+
+  deleteAdmin(username: string) {
+    getCollection("admin").deleteOne({username});
```

Comment:
We should probably check `deletedCount` to make sure that the user was really deleted

Reply:
You might add that yourself as I'm to stupid understand what you're trying to achieve

Reply:
Added by me